packages/@okta/vuepress-site/docs/guides/mobile-idx-sdk-overview/main/ios/loadingaconfiguration.md

```diff
@@ -0,0 +1,60 @@
+Initialize a configuration by passing the appropriate values to `IDXClient.Configuration(issuer:clientId:clientSecret:scopes:redirectUri:)`. You can set the values directly in your code or load them from a file. This example shows loading the values from a property file in your project.
```

Comment:
"You can set the values directly in your code or load them from a file" 
This repeats a sentence in the para immediately above here in index.md. You'll need to delete this or the one above.